extensions/ql-vscode/src/interface.ts

```diff
@@ -363,13 +363,28 @@ export class InterfaceManager extends DisposableObject {
       });
     }
 
+    let resultSets: RawResultSet[] | undefined;
+
+    if (EXPERIMENTAL_BQRS_SETTING.getValue()) {
+      resultSets = [];
+      const pageSize = 100;
+      const schemas = await this.cliServer.bqrsInfo(results.query.resultsPaths.resultsPath, pageSize);
+      for (const schema of schemas["result-sets"]) {
+        const chunk = await this.cliServer.bqrsDecode(results.query.resultsPaths.resultsPath, schema.name, pageSize, 0)
```

Comment:
Does this get only the first 100 results for each resultSet? Maybe ok for a PoC. Though, I'd be leaning towards getting all of the results for the PoC just so we can see how much juice these codespace vms really have.

Reply:
Ah, sure, good point.